### PR TITLE
A new partition scheme for RHEL 8

### PR DESCRIPTION
**Pull request by neo954**
### The PR is for implement feature https://github.com/xcat2/xcat2-task-management/issues/596

Here is the new partition scheme for RHEL 8.

**For disk drive less than 60GiB**

| File system | Size | Comment |
|----|----|----|
| /boot | 1024MiB | As recommended by Red Hat installation manual |
| swap | Recommended size | LVM logical volume |
| / | 86% disk size | LVM logical volume |

**For disk drive equals or larger than 60GiB**

| File system | Size | Comment |
|----|----|----|
| /boot | 1024MiB| As recommended by Red Hat installation manual |
| swap | Recommended size | LVM logical volume |
| / | 10240MiB - 20480MiB | LVM logical volume |
| /var | 4096MiB - 8192MiB | LVM logical volume |
| /tmp| 4096MiB - 8192MiB | LVM logical volume |
| /home | 10% disk size | LVM logical volume |

### CODE REVIEW

**Comment by immarvin** [comment]
hi @whowutwut , do you think the partition scheme is ok?

File: xCAT-server/share/xcat/install/scripts/pre.rhels8 [hunk 1]
```diff
@@ -242,16 +268,24 @@ then
...
+echo "part /boot --fstype=$BOOTFSTYPE --asprimary --ondisk=$instdisk --size=1024" >>/tmp/partitionfile
 echo "part pv.000997 --grow --asprimary --ondisk=$instdisk --size=18432" >>/tmp/partitionfile
 echo "volgroup xcatvg --pesize=4096 pv.000997" >>/tmp/partitionfile
-echo "logvol / --fstype=$FSTYPE --name=root --vgname=xcatvg --grow --size=4096 --maxsize=20480" >>/tmp/partitionfile
-echo "logvol /var --fstype=$FSTYPE --name=var --vgname=xcatvg --grow --size=2048 --maxsize=8192" >>/tmp/partitionfile
-echo "logvol /tmp --fstype=$FSTYPE --name=tmp --vgname=xcatvg --grow --size=1024 --maxsize=4096" >>/tmp/partitionfile
-echo "logvol /home --fstype=$FSTYPE --name=home --vgname=xcatvg --grow --percent=10 --maxsize=10240" >>/tmp/partitionfile
 echo "logvol swap --name=swap --vgname=xcatvg --recommended" >>/tmp/partitionfile
 
+if [ "${DISKSIZE}" -lt "62914560" ]
```
Comment (robin2008) [inline xCAT-server/share/xcat/install/scripts/pre.rhels8]: what's behavior when `DISKSIZE` is empty (assume `getdisksize` hit some errors)

**Comment by immarvin** [comment]
I am ok with the change, agree to merge, thanks @neo954

**Comment by immarvin** [comment]
@neo954  will create a ticket to describe the new partition scheme explicitly in RTD

**Pull request merged**